tools/integration_tests/util/setup/setup.go

```diff
@@ -28,6 +42,15 @@ var (
 	mntDir  string
 )
 
+// Run shell script
+func RunScriptForTestData(script string, testBucket string) {
+	cmd := exec.Command("/bin/bash", script, testBucket)
+	_, err := cmd.Output()
+	if err != nil {
+		panic(err)
+	}
+}
+
 func TestBucket() string {
```

Comment:
Let's call this method everywhere, we are referring *testBucket or *mountedDirectory.

Reply:
Swetha suggested to not use getter setter in the same file for variables. In outer files I used these functions only.